src/tests/ftest/io/ior_hdf5.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/python
+"""
+(C) Copyright 2018-2019 Intel Corporation.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+The Government's rights to use, modify, reproduce, release, perform, display,
+or disclose this software are subject to the terms of the Apache License as
+provided in Contract No. B609815.
+Any reproduction of computer software, computer software documentation, or
+portions thereof marked with this legend must also reproduce the markings.
+"""
+
+from ior_test_base import IorTestBase
+
+
+class IorHdf5(IorTestBase):
+    # pylint: disable=too-few-public-methods
+    """Test class Description: Runs IOR/HDF5 on 1 server with basic parameters.
+
+    :avocado: recursive
+    """
+
+    def ior_hdf5(self, hdf5_plugin_path=None):
```

Comment:
Shouldn't this be an `IorTestBase` method?  I can think of at least one test case that would benefit from having this common.  If it is moved to `IorTestBase`, then the three values obtained from the test yaml should be method inputs.